bin/fzf-preview.sh

```diff
@@ -57,7 +57,12 @@ if [[ $KITTY_WINDOW_ID ]]; then
 
 # 2. Use chafa with Sixel output
 elif command -v chafa > /dev/null; then
-  chafa -f sixel -s "$dim" "$file"
+  # Alacritty doesn't support sixels
+  if [[ $ALACRITTY_WINDOW_ID ]]; then
+    chafa --colors 256 -s "$dim" "$file"
+  else
+    chafa -f sixel -s "$dim" "$file"
+  fi  
```

Comment:
The man page of chafa states that it auto-detects the right format.

```
OUTPUT ENCODING
       -f, --format format
           Set output format; one of [iterm, kitty, sixels, symbols]. The
           default is iterm, kitty or sixels if the connected terminal
           supports one of these, falling back to symbols ("ANSI art")
           otherwise.

```

Which means we can just drop `-f sixel` instead?

```suggestion
  chafa -s "$dim" "$file"
```

---

Hmm, no, chafa fails to detect it's on iTerm2 and doesn't render image nicely on it. The issue is fixed on master, but no releases yet.

https://github.com/hpjansson/chafa/issues/196

---

I don't use alacritty, but I believe it's capable of showing more than 256 colors. So why don't we just drop `--colors 256`?

```
If left unspecified, an optimal default will be chosen based on the current environment.
```

Reply:
I'll give this a test ... on vacation for a week but have some time today to verify

Reply:
<img width="740" alt="image" src="https://github.com/junegunn/fzf/assets/6491743/4de5bbdc-2f29-4816-a722-5524a6d9407d">

I can confirm this ... on iterm2 it shows up as ansi/

<img width="119" alt="image" src="https://github.com/junegunn/fzf/assets/6491743/8e2328ca-5d25-40f2-9664-28f21b40b648">

Reply:
Now that we have the new chafa version with the fix, I think we can just drop `-f sixel` as suggested above.